### DISCUSSION TITLE
Update policy for Ruby gems in gsd-tools Gemfile files

### DISCUSSION THREAD

**Discussion by jasnow** [cloudsecurityalliance/gsd-tools#227] [General]
What is the update policy for Ruby gems in gsd-tools Gemfile files?

Also are you participating in the [Hacktoberfest 2023](https://hacktoberfest.com) program this month?

**Comment by joshbuker** [comment]
Could you expand on what you mean when you say update policy? I don't understand the question.

**Reply by jasnow** [reply]
After finding 3 Gemfile files, I ran "bundle outdated" on each (see
below). As the third one is data and the second one's gems are
up-to-date, and the 1st one does not have test, I think the 
best course for me is to close this question.
Take care.

======================================================================
++ cd /home/t530-dev/Projects/gsd-tools/website
++ bundle outdated
Fetching gem metadata from https://rubygems.org/............
Resolving dependencies...

Gem                    Current  Latest  Requested  Groups
ffi                    1.15.5   1.16.3
google-protobuf        3.24.3   3.24.4
liquid                 4.0.4    5.4.0
sass-embedded          1.68.0   1.69.4
unicode-display_width  2.4.2    2.5.0
======================================================================
++ cd /home/t530-dev/Projects/gsd-tools/gsd-libraries/ruby/gsd-database
++ bundle outdated
Resolving dependencies...

Bundle up to date!
======================================================================
++ cd /home/t530-dev/Projects/gsd-tools/gsd-importers/ruby-advisory-db
++ bundle outdated
Could not find gem 'git' in locally installed gems.
======================================================================
EOF
